bin/merge_indices.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Feb 16 12:00:04 2024
+
+@author: madeline
+
+This script merges multiple mutation indices into one.
+If an existing mutation index is provided, the new indices are added to it.
+"""
+
+import dask.dataframe as dd
+import pandas as pd
+import argparse
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Merges mutation indices into one.')
+    parser.add_argument('--original_index', type=str, default=None,
+                        help='Path to an existing TSV index of all mutations, \
+                            to merge new indices to')
+    parser.add_argument('--gvf_indices', type=str, default=None,
+                        nargs='*', help='Paths to GVF indices to merge')
+    parser.add_argument('--index_savefile', type=str,
+                        default=None, help='TSV filename to save updated \
+                            index of all mutations to')
+
+    return parser.parse_args()
+
+
+if __name__ == '__main__':
+
+    args = parse_args()
+    original_index = args.original_index
+    gvf_indices_list = args.gvf_indices
+    index_savefile = args.index_savefile
+
+    # if an original index is provided, merge the new indices with it
+    if original_index!=None:
+        indices_to_merge = list(original_index) + gvf_indices_list
+    # if an original index is not provided, make an entirely new index
+    else:
+        indices_to_merge = gvf_indices_list
+    
+    # read all indices into dask df
+    ddf = dd.read_csv(indices_to_merge, sep='\t') 
+    # fillna to make groupby() work
+    ddf = ddf.fillna('n/a')
+    # specify which columns to group by
+    group_cols = [x for x in ddf.columns if x!='lineages']
+    '''
+    # specify meta df: this throws an error, but the inferred meta works fine
+    schema = {**dict.fromkeys(ddf.columns, 'str'), 'pos':'int'}
+    meta_df = pd.DataFrame(columns=schema.keys()).astype(schema)
```

Comment:
This specific line uses pandas and if I am not wrong, pandas are not used anywhere else in this script. Is it possible to convert pandas to dask here too? This should help us with containerization. 
The current version of dask is [2023.10.1-py11-ol9_cv1](https://hub.docker.com/layers/biocontainers/dask/2023.10.1-py11-ol9_cv1/images/sha256-8bd5e6fad3a0c3e8009edab4217662b8c5b56bc11a8d2305f3b707509299dd71?context=explore). Can you please try with this version?

Reply:
deleted the use of `meta` here and it works fine (though would be nice to specify someday)